releasenotes/notes/varlogscontainers-ff-670c610835a52741.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    With Kubernetes, use `logs_config.validate_pod_container_id` to use symlinks in
+    `/var/log/containers` to validate the container ID tag while parsing pod logs.
```

Comment:
since this is about a feature that hasn't been released yet, let's update the release note of the initial PR (#8143) 
 instead of creating a new one